.github/ISSUE_TEMPLATE/installation_issue_template.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Installation Issues
+about: Use this template for reporting bugs encountered while installing MLflow.
+labels: 'bug'
+title: [SETUP-BUG]
+---
+Thank you for submitting an issue. Please refer to our [issue policy](https://www.github.com/mlflow/mlflow/blob/master/ISSUE_POLICY.md)
+for information on what types of issues we address.
+  
+Please do not delete this template unless you are sure your issue is outside its scope.
+
+### System information
+- **Have I written custom code (as opposed to using a stock example script provided in MLflow)**:
```

Comment:
I think we can remove the custom code question? Doesn't seem as relevant for installation issues